docs/source/java/substrait.rst

```diff
@@ -0,0 +1,543 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+=========
+Substrait
+=========
+
+Cross-Language Serialization for Relational Algebra.
+
+`Substrait`_ is a well-defined, cross-language specification for data compute operations.
+
+.. contents::
+
+Getting Started
+===============
+
+There are two perspectives among substrait implementors:
+
+- Substrait Consumer
+- Substrait Producer
+
+Substrait is calling Acero thru JNI wrappers to Consume Substrait plans.
+
+.. seealso:: :doc:`../cpp/streaming_execution` for more information on Acero.
```

Comment:
I think the docs need to be rewritten. Acero is introduced out of nowhere, and it's unclear whether this is trying to be a Substrait reference or an Arrow reference.

Reply:
If the latter, just state that the Java JNI modules provide a Substrait consumer based on Acero, and link to any supporting documentation as needed. We are not the Substrait project.